src/Storage/Controllers/InstancesController.cs

```diff
@@ -872,28 +801,13 @@ private string GetUserId()
             return User.GetUserOrOrgId();
         }
 
-        private string BuildRequestLink(string continuationToken = null, Dictionary<string, StringValues> queryParams = null)
+        private string BuildRequestLink(InstanceQueryParameters queryParams)
```

Comment:
I still don't understand why you can't keep the original BuildRequestLink (with the correct cotinueationtoken passed)

Reply:
The methods is now uses this function to build the continuation token.